### ISSUE TITLE
Vendure-v3.04 not switching between dark and light mode

### ISSUE THREAD

**Issue by ashishkpaul** [vendure-ecommerce/vendure#3111]
**Describe the bug**
![Screenshot_2024-10-06_09-16-16](https://github.com/user-attachments/assets/82f54e90-cb50-4900-83be-af06493ed243)

**To Reproduce**
1. Go to 'Dashboard'
2. Click on 'Theme'

**Expected behavior**

**Environment (please complete the following information):**
 - @vendure/core version:
 - Nodejs version
 - Database (mysql/postgres etc):

**Additional context**

**Comment by roberttolton** [comment]
This still seems broken in 3.0.5?

Fresh install as of the start of October, updated to 3.0.5 today via updating dependencies and `npm install`.

https://github.com/user-attachments/assets/e7f9f50f-8d3b-4600-8be8-8052d7e4f83c

**Comment by michaelbromley** [comment]
@roberttolton can you try:

1. deleting any `admin-ui` dir created by the compilation process
2. delete any `.angular` dir that may exist
3. delete node_modules & lockfile, reinstall dependencies

and try again and see if that resolves it?

**Comment by roberttolton** [comment]
Hi @michaelbromley, thanks for your help.

This morning I installed a fresh 3.0.5 to compare against, and saw that a bunch of the CSS variables were getting overridden. It hit me then that I had began to customise the theme weeks ago before I had to pause work and come back to it now, and completely forgot.

For my custom theme, I'm now making sure it targets a specific mode e.g `:root [cds-theme=light]` and everything works.

Apologies for wasting your time, I am certainly banging my head on the desk now!
